src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -15,20 +15,23 @@
 -->
 
 <kd-help-section>
-  <md-input-container class="md-block" md-is-error="ctrl.isNameError()">
+  <md-input-container class="kd-deploy-block" md-is-error="ctrl.isNameError()">
```

Comment:
If this is the only place where it is ignored then adjust this one places and check for other places. It can't be ignored in any place where it is used.